wallet/src/ui/app/components/transactions-card/index.tsx

```diff
@@ -0,0 +1,96 @@
+// Copyright (c) 2022, Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import cl from 'classnames';
+
+import AccountAddress from '_components/account-address';
+import ExplorerLink from '_components/explorer-link';
+import { ExplorerLinkType } from '_components/explorer-link/ExplorerLinkType';
+
+import type { ExecutionStatusType, TransactionKindName } from '@mysten/sui.js';
+
+import st from './TransactionsCard.module.scss';
+
+type TxnData = {
+    To?: string;
+    seq: number;
+    txId: string;
+    status: ExecutionStatusType;
+    txGas: number;
+    kind: TransactionKindName | undefined;
+    From: string;
+};
+const truncate = (txt: string, maxLength: number) => {
+    if (txt.length < maxLength + 3) {
+        return txt;
+    }
+    const beginningLength = Math.ceil(maxLength / 2);
+    const endingLength = maxLength - beginningLength;
+    return `${txt.substring(0, beginningLength)}...${txt.substring(
+        txt.length - endingLength
+    )}`;
+};
+
+function TransactionResult({ txresults }: { txresults: TxnData[] }) {
+    return (
+        <div className={st['tx-container']}>
```

Comment:
After [this](https://github.com/MystenLabs/sui/commit/a7ec54bed3476de76c1d72779d343ea55261162e#diff-0d73a2b9d22dfdd04ea4dd097a5cdb125ed232bd9f164074bce9f2af787bd69bR115) all class names exported from modules will be in camel case as well eg. `txContainer`. I think it makes it a bit easier to use. (At some point we can refactor all usages and then export only the camelCase format)